docs/docs/kds-rest-api.md

```diff
@@ -2,21 +2,21 @@
 
 ## Introduction
 
-This page describes how applications can communicate with the Key Derivation Service, also known as KDS. The service is made accessible
-through its REST API endpoints which are presented below. As many applications, KDS also uses JSON to send and receive data.
+This page describes how applications can communicate with the Key Derivation Service (KDS). The KDS uses JSON to send 
+and receive data. You can access the KDS through its REST API endpoints given below.
```

Comment:
Not sure why the original author felt the need to say that the KDS uses json... Kinda like saying that rain contains water! If the goal of the sentence is to let the user know what type of application the KDS is, then I'd maybe replace the second two sentences with something like:

"The KDS is a web-server which implements a REST API for deriving public and private keys."